scripts/makebumpver

```diff
@@ -121,8 +128,7 @@ class JIRAValidator:
     def validate_RHEL_issue_fixversion(self, bug):
         issue = self._query_RHEL_issue(bug)
 
-        # TODO: Switch this to template solution "distro_release"
-        valid_fix_version = "rhel-9"
+        valid_fix_version = rhel-9
```

Comment:
- How about adding a small helper module, templating only that, and importing the value from there? Working on templated code is not nice.
- `"rhel-9"` -> `rhel-9` means you had a string but replaced it with nonexistent variable minus the number 9

Reply:
I think this is something we can discuss more. Might be beneficial, however, I would avoid to introduce this new approach on RHEL branch.

Reply:
I was thinking a bit more about it and we already do that with KS version and it will simplify things. Do you agree with current approach or should I move the module somewhere else?

Reply:
I think it's fine - let's tune details later, if needed.